unlock-app/src/utils/withReduxStore.js

```diff
@@ -40,11 +40,13 @@ export default (App) => {
       }
     }
 
+    /**
+     * initialReduxState is actually coming from the server side rendered content
+     * We want to "merge" it with the data that might be stored in localStorage
+     */
     constructor (props) {
       super(props)
-
-      // For some reason this argument is being ignored... we probably need to use it!
-      this.reduxStore = getOrCreateStore(props.initialReduxState)
+      this.reduxStore = getOrCreateStore(Object.assign(props.initialReduxState, loadState()))
```

Comment:
I like the symmetry of having loadState() and getState() be in the same code block!